### ISSUE TITLE
[Comprehension] User doesn't have enough information to complete a task.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#75986]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** govcio-vfep
**Product:** GI Bill Comparison Tool
**Feature:** Specialized Mission Filter

## Findings details
**VA.gov Experience Standard - issue:** User doesn't have enough information to complete a task.
**VA.gov Experience Standard - category:** Comprehension
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the "Search by location" tab, the distance `select` doesn't have a visible label. It does have an `aria-label="Distance"`, which provides an accessible name for screen reader users. But whenever possible on VA.gov we try to have visible text labels on form inputs. Visible text labels on form inputs help in two big ways: 1. Reduce cognitive load for sighted users. No one has to guess what a form input is for. 2. Provide clear labels for voice command users. Users of software like Dragon NaturallySpeaking typically select a form field by saying "Click [name of field]," eg. "Click City, state, or postal code." When there's not a visible label, they have to guess what the magic words are to select that field, and may guess wrong (like "Click within 25 miles," which won't generate a match).

## Recommended action
Remove the `aria-label` and just have a visible `label` for that `select`.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
